src/render.zig

```diff
@@ -366,15 +366,15 @@ pub const Renderer = struct {
     /// Get the drawing area for the current target.
     pub fn getViewport(
         self: Renderer,
-    ) ?rect.Rect {
+    ) !rect.IRect {
         var viewport: C.SDL_Rect = undefined;
         const ret = C.SDL_GetRenderViewport(
             self.value,
             &viewport,
         );
         if (!ret)
             return error.SdlError;
-        return rect.Rect.fromSdl(viewport);
+        return rect.IRect.fromSdl(viewport);
```

Comment:
Funny enough I found this and some more in my local branch finishing up render lol. Keep this in though, I'll rebase when it's time. Thank you for your changes!